builds/e2e/longhaul.yaml

```diff
@@ -346,4 +346,128 @@ jobs:
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
           loadGen.message.frequency: '$(loadGen.message.frequency.amd64)'
-         
\ No newline at end of file
+          
+################################################################################
+  - job: setup_iotuap_amd64_moby
+################################################################################
+    displayName: Windows IoT Core AMD64 Moby (Setup Only)
+    condition: eq(variables['run.iotuap.amd64.moby'], 'true')
+    pool:
+      name: $(pool.name)
+      demands:
+        - longhaul_iotuap_x64 -equals true
+    variables:
+      edgelet.artifact.name: 'iotedged-windows'
+    steps:
+      - checkout: none
+      - task: AzureKeyVault@1
+        displayName: 'Azure Key Vault'
+        inputs:
+          azureSubscription: $(azure.subscription)
+          KeyVaultName: $(azure.keyVault)
+          SecretsFilter: 'edgebuilds-azurecr-io-username,edgebuilds-azurecr-io-pwd,IotHubStressConnString,EventHubStressConnStr,StorageAccountMasterKeyStress,SnitchLongHaulAlertUrl'
+      - task: DownloadBuildArtifacts@0
+        displayName: 'Download Edgelet Artifacts'
```

Comment:
Please work with Chanisha to refactor download steps as template for reuse.  I tried to put azure key vault step into template before, but I remember this may cause some credential issue, you can try on yourself to see if it works now.

Reply:
Let’s work on a template for reuse across all Windows editions, but in a separate code review and keep this one focused on Windows IoT Core - adding the missing Stress and Longhaul tests to get on par with Windows Enterprise and Windows Server (all x64).